ReactiveCocoa.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ReactiveCocoa'
-  s.version = '4.0.2-alpha-3'
+  s.version = '4.0.3-alpha-3'
```

Comment:
We're currently on `4.0.4-alpha-1` on CP

https://cocoapods.org/pods/ReactiveCocoa